readthedocs-theme/templates/readthedocs/company.html

```diff
@@ -161,14 +161,15 @@ <h3 class="ui header"><small><i class="icon heart"></i></small> Contributors</h3
           <div class="ui segment vertical padded">
             <div class="ui grid stackable">
               <div class="five wide column">
-                <a class="ui avatar" href=""><img class="ui avatar image" src=""></a>
-                <a class="ui avatar" href=""><img class="ui avatar image" src=""></a>
-                <a class="ui avatar" href=""><img class="ui avatar image" src=""></a>
-                <a class="ui avatar" href=""><img class="ui avatar image" src=""></a>
-                <a class="ui avatar" href=""><img class="ui avatar image" src=""></a>
-                <a class="ui avatar" href=""><img class="ui avatar image" src=""></a>
-                <a class="ui avatar" href=""><img class="ui avatar image" src=""></a>
-                <a class="ui avatar" href=""><img class="ui avatar image" src=""></a>
+                <a class="ui avatar" href="https://github.com/Blendify" title="Blendify"><img class="ui avatar image" src="/theme/img/contributors/Blendify.jpg"></a>
+                <a class="ui avatar" href="https://github.com/jessetan" title="jessetan"><img class="ui avatar image" src="/theme/img/contributors/jessetan.jpg"></a>
+                <a class="ui avatar" href="https://github.com/saadmk11" title="saadmk11"><img class="ui avatar image" src="/theme/img/contributors/saadmk11.jpg"></a>
+                <a class="ui avatar" href="https://github.com/dojutsu-user" title="dojutsu-user"><img class="ui avatar image" src="/theme/img/contributors/dojutsu-user.jpg"></a>
+                <a class="ui avatar" href="https://github.com/safwanrahman" title="safwanrahman"><img class="ui avatar image" src="/theme/img/contributors/safwanrahman.jpg"></a>
+                <a class="ui avatar" href="https://github.com/tapaswenipathak" title="tapaswenipathak"><img class="ui avatar image" src="/theme/img/contributors/tapaswenipathak.jpg"></a>
+                <a class="ui avatar" href="https://github.com/cocobennett" title="cocobennett"><img class="ui avatar image" src="/theme/img/contributors/cocobennett.jpg"></a>
+                <a class="ui avatar" href="https://github.com/Blackcipher101" title="Blackcipher101"><img class="ui avatar image" src="/theme/img/contributors/Blackcipher101.jpg"></a>
+                <a class="ui avatar" href="https://github.com/Abhi-khandelwal" title="Abhi-khandelwal"><img class="ui avatar image" src="/theme/img/contributors/Abhi-khandelwal.jpg"></a>
```

Comment:
The spacing on these images looks a bit off, I think becase `ui avatar` usually has inline text beside it.

`ui images` doesn't and probably has a better layout. Centering the content may help but i'm not decided.

```html
              <div class="five wide center aligned column">
                <div class="ui mini circular images">
                  <a class="" href="https://github.com/Blendify" title="Blendify"><img class="ui image" src="/theme/img/contributors/Blendify.jpg"></a>
                  <a class="" href="https://github.com/jessetan" title="jessetan"><img class="ui image" src="/theme/img/contributors/jessetan.jpg"></a>
                  <a class="" href="https://github.com/saadmk11" title="saadmk11"><img class="ui image" src="/theme/img/contributors/saadmk11.jpg"></a>
                  <a class="" href="https://github.com/dojutsu-user" title="dojutsu-user"><img class="ui image" src="/theme/img/contributors/dojutsu-user.jpg"></a>
                </div>
                <div class="ui mini circular images">
                  <a class="" href="https://github.com/safwanrahman" title="safwanrahman"><img class="ui image" src="/theme/img/contributors/safwanrahman.jpg"></a>
                  <a class="" href="https://github.com/tapaswenipathak" title="tapaswenipathak"><img class="ui image" src="/theme/img/contributors/tapaswenipathak.jpg"></a>
                  <a class="" href="https://github.com/cocobennett" title="cocobennett"><img class="ui image" src="/theme/img/contributors/cocobennett.jpg"></a>
                  <a class="" href="https://github.com/Blackcipher101" title="Blackcipher101"><img class="ui image" src="/theme/img/contributors/Blackcipher101.jpg"></a>
                  <a class="" href="https://github.com/Abhi-khandelwal" title="Abhi-khandelwal"><img class="ui image" src="/theme/img/contributors/Abhi-khandelwal.jpg"></a>
                </div>
              </div>
```

The additional spacing at the the top of the box could be better filled too:
![image](https://user-images.githubusercontent.com/1140183/154220050-369c90f1-85fb-499a-866f-85fa0587970e.png)

Reply:
👍  I personally prefer images to be left aligned, so the heading above is not left hanging and is also more in line with the content around:

![Screenshot 2022-02-16 103314](https://user-images.githubusercontent.com/4049894/154246744-58058fda-88ac-4ecf-8957-1e9d8d83da57.png)

```html
<div class="ui grid relaxed stackable">
  <div class="five wide column">
    <div class="ui mini circular images">
      <a href="https://github.com/Blendify" title="Blendify"><img class="ui image" src="/theme/img/contributors/Blendify.jpg"></a>
      <a href="https://github.com/jessetan" title="jessetan"><img class="ui image" src="/theme/img/contributors/jessetan.jpg"></a>
      <a href="https://github.com/saadmk11" title="saadmk11"><img class="ui image" src="/theme/img/contributors/saadmk11.jpg"></a>
      <a href="https://github.com/dojutsu-user" title="dojutsu-user"><img class="ui image" src="/theme/img/contributors/dojutsu-user.jpg"></a>
      <a href="https://github.com/safwanrahman" title="safwanrahman"><img class="ui image" src="/theme/img/contributors/safwanrahman.jpg"></a>
      <a href="https://github.com/tapaswenipathak" title="tapaswenipathak"><img class="ui image" src="/theme/img/contributors/tapaswenipathak.jpg"></a>
      <a href="https://github.com/cocobennett" title="cocobennett"><img class="ui image" src="/theme/img/contributors/cocobennett.jpg"></a>
      <a href="https://github.com/Blackcipher101" title="Blackcipher101"><img class="ui image" src="/theme/img/contributors/Blackcipher101.jpg"></a>
      <a href="https://github.com/Abhi-khandelwal" title="Abhi-khandelwal"><img class="ui image" src="/theme/img/contributors/Abhi-khandelwal.jpg"></a>
    </div>
  </div>
  <div class="eleven wide column">
      ...
  </div>
</div>
```

Reply:
Perhaps lets trim one off so we can have even rows then. It looks unfinished as staggered rows.

Also, I would be on board with overlapping avatars:

![image](https://user-images.githubusercontent.com/1140183/154333569-e5c1ae5a-6e6c-4eab-bebf-8e96591fd876.png)

This is something that we'd reuse on RTD dashboard as well, to list maintainers or project teams, or project team members. It might need to be a little custom, but I'm sure we could extend `ui images` pretty easily to use a negative left margin or similar.

Perhaps then, even out the rows and implement overlapping circles in ext-theme to start?

Reply:
https://github.com/readthedocs/site-community/pull/72/commits/150efa432f8eb89c131ab244cbe54adfe94a6b83 adds `.overlapping.images`.

![Screenshot 2022-02-16 192019](https://user-images.githubusercontent.com/4049894/154340072-eb207889-fbd4-4b43-b7bf-dd09bbf9e084.png)

I can also add a circle with the total number of contributors, linking to _readthedocs.org_ repo contributors page, but would have to be custom styled from scratch, and as its a big number, it would probably not look so good inside a circle...

Reply:
Hah yeah, this is where a `ui circular label` or something might fit at the end, but I have similar concerns. Looks good!